src/test/java/duke/logic/autocorrect/AutocorrectTest.java

```diff
@@ -0,0 +1,30 @@
+package duke.logic.autocorrect;
+
+import duke.commons.exceptions.DukeException;
+import duke.storage.Storage;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class AutocorrectTest {
+    private Storage storage = new Storage();
+    private Autocorrect autocorrect = new Autocorrect();
+
+    @Test
+    void autocorrectTest() {
+        try {
+            storage.loadWord(autocorrect);
+        } catch (DukeException e) {
+            System.out.println("Error");
+        }
+        autocorrect.setWord("calorei");
```

Comment:
you should be testing the method that is called within execute to unit test this.